packages/sit-onyx/src/components/OnyxListbox/types.ts

```diff
@@ -1,22 +1,29 @@
-import type { SelectionOptionValue } from "../OnyxRadioButton/types";
+import type { ListboxValue, ListboxModelValue } from "@sit-onyx/headless";
 
-export type OnyxListboxProps<TValue extends SelectionOptionValue = SelectionOptionValue> = {
+export type OnyxListboxProps<
+  TMultiple extends boolean = false,
+  TOption extends ListboxValue = ListboxValue,
```

Comment:
Also sometimes TMultiple is the first generic, sometimes its TOption.
Would prefer to list TOption/TValue first for consistancy

Reply:
I used TOption as name instead of TValue because in all other places I saw, TValue immediatly is the type of modelValue

Reply:
I reorderd it. is it ok to keep the `TOption` name because of above reason?

Reply:
The reason for naming it `TValue` is that the types is used for the option value (which is the case for the modelValue property but also when passing it to the options) so imho `TOption` would give the wrong impression here :)

Otherwise it might be confusing that a type called `TOption` is restricted to `string | number | boolean`

Reply:
done!